src/components/modals/item-modal/PlaceBidModal.tsx

```diff
@@ -31,6 +31,11 @@ const PlaceBidModal = ({ show, handleModal, item }: Props) => {
           listId: item.listId,
         })
       );
+
+      if (type.includes('fulfilled')) {
```

Comment:
I'd suggest to not doing this logic here. Redux toolkit handles this perfectly for us through the reducers (the normal way of redux). You could make the dispatch here and get the type from a selector very easily and leverage the logic to the reducers.

Reply:
Ok, then I have to move all of this to an action